documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -29,23 +29,3 @@ endif::[]
 [kni@provisioner ~]$ mkdir ~/clusterconfigs
 [kni@provisioner ~]$ cp install-config.yaml ~/clusterconfigs
```

Comment:
This piece right here doesn't have the correct numbering.

Reply:
Good catch, I'll probably have to include the header in the configuring the install config instead of the individual ones

Reply:
Solved in latest push